### ISSUE TITLE
Backport #189 prior to Quarkus 3.0.0 Update

### ISSUE THREAD

**Issue by Nplu5** [quarkiverse/quarkus-jberet#206]
Would it be possible to release the changes from #189 as a patch for the version of this extension before 2.0.0? E.g. a release of a 1.2.2 as we would need the change but did not and cannot yet migrate to Quarkus 3.0.0?

**Comment by radcortez** [comment]
Can you provide me with a branch with the backport for 1.2.2?

**Comment by Nplu5** [comment]
Yes will do! I'll branch of the 1.2.1 Tag or should I branch of from a later commit?

**Comment by Nplu5** [comment]
Prepared the branch in my fork. To which branch should I create the pull-request? 

Backport branch: https://github.com/Nplu5/quarkus-jberet/tree/thread-configuration-backport

**Comment by Nplu5** [comment]
@radcortez just wanted to check to which branch I should create the pull-request?

**Comment by radcortez** [comment]
Please, use https://github.com/quarkiverse/quarkus-jberet/tree/1.x. Thanl you.
